css/uswds-together.scss

```
@@ -35,8 +34,7 @@ $next-theme-colors: (
   "next-blue-medium" #2672de,
   "next-blue-dark" #1f5aad,
   "next-mint-medium" #74d7ad true,
-  "next-mint-dark" #45a17a true
-  "together-green" #00A91C true,
+  "next-mint-dark" #45a17a true "together-green" #00a91c true
```

Looks like we're missing a comma to separate these two values.

```suggestion
  "next-mint-dark" #45a17a true,
   "together-green" #00a91c true
```